crates/consensus/beacon/src/engine/mod.rs

```diff
@@ -233,6 +236,8 @@ where
     invalid_headers: InvalidHeaderCache,
     /// Consensus engine metrics.
     metrics: EngineMetrics,
+    /// The largest gap for which the tree will be used to download and execute ranges of blocks.
```

Comment:
should provide the context of what "gap" means here: block distance from FCU head to local hea

Reply:
added more context